utils/pkg/pkg_run

```diff
@@ -158,9 +162,22 @@ package_cloud_push () {
       export PC_REPO="worker-testing"
     fi
 
-    package_cloud push \
-      $PC_USER/$PC_REPO/$package_cloud_os/$platform_release \
-      tmp/output/$pkgtype/$platform_family/$platform_release/*.$pkgtype --verbose
+    PC_UPLOAD_FILE=$(ls $CHECKOUT_ROOT/tmp/output/$pkgtype/$platform_family/$platform_release/*.$pkgtype)
```

Comment:
The @tianon in the back of my head says using `ls` here is asking for trouble :trollface: